README.md

```diff
@@ -89,16 +97,16 @@ Besides the normal connection parameters that are described above this function
 
 ```dart
 final characteristic = QualifiedCharacteristic(serviceId: serviceUuid, characteristicId: characteristicUuid, deviceId: foundDeviceId);
-final response = await reactiveBleClient.readCharacteristic(characteristic);
+final response = await flutterReactiveBle.readCharacteristic(characteristic);
 ```
 
 #### Write with response
 
-Write a value to characteristic and await the response. The return value is a list of bytes (`List<int>`).
+Write a value to characteristic and await the response. The return value is a `Future<void>`, since BLE does not support a request/response mechanism like for example Http.
```

Comment:
Mentioning request-response mechanism from HTTP and not explaining what the `-Response` part _actually_ means here will leave a user with a bit of confusion, I expect.

Also, please, capitalize abbreviations: HTTP.